{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Welcome to my application project. Here, I analyze a data set in order to get my first hands-on experience with single-cell analysis.\n",
    "After talkigng to Pau, he recommended some twitter posts in which I could find data sets.\n",
    "\n",
    "I researched a paper called \"Single-cell genomics identifiescell type–specific molecularchanges in autism\" by Dmitry Velmeshev et al. from 2019.\n",
    "Citation: \"Dmitry Velmeshev et al. ,Single-cell genomics identifies cell type–specific molecular changes in autism. Science 364, 685-689(2019). DOI:10.1126/science.aav8130\"\n",
    "\n",
    "My plan is to analyze the dataset and compare my results alongside the paper as a reference.\n",
    "\n",
    "About the data set:\n",
    "\n",
    "\n",
    "Challenges:\n",
    "- Clinical and genetic heterogenity of autism\n",
    "- Feasability of direct assessment of specific cell types in the brain affected by autism (until recently)\n",
    "\n",
    "\n",
    "\n",
    "Samples:\n",
    "- 41 postmortem tissue samples including prefrontal cortex (PFC) and anterior cingulate cortex (ACC) from 15 ASD patients and 16 controls\n",
    "- Samples in the control and ASD groups were between 4 and 22 years old and were matched for age, sex, RNA integrity number, and postmortem interval (fig. S1A; P > 0.1, Mann-Whitney U test)\n",
    "- None of the ASD patients were diagnosed with intellectual disability; however, half of the ASD patients had a history of seizures, a common comorbidity in ASD\n",
    "\n",
    "########################################\n",
    "Methods:\n",
    "- Single-nucleus RNA sequencing (snRNA-seq) of of cortical tissue from patients with autism to identify autism-associated transcriptomic changes in specific cell types\n",
    "- To compare changes in ASD to those in patients with sporadic epilepsy only, we generated additional snRNA-seq data from eight PFC samples from patients with sporadic epilepsy and seven age-matched controls (data S1)\n",
    "\n",
    "####################\n",
    "SAMPLES:\n",
    "- We processed tissue samples for nuclei isolation and snRNA-seq using the 10× Genomics platform (Fig. 1B).\n",
    "- We generated 104,559 single-nuclei gene expression profiles—52,556 from control subjects and 52,003 from ASD patients (data S2)—and detected a median of 1391 genes and 2213 transcripts per nucleus, a yield that is in agreement with a recent snRNA-seq study (5). Numbers of genes and transcripts were uniform between control and ASD samples (fig. S1B)\n",
    "- Additionally, we generated 21,984 single-nucleus profiles sampled from patients with epilepsy and controls.\n",
    "- The multiplet nuclei capture rate was comparable to single-cell RNA-seq analysis using the 10× platform (fig. S1C). The libraries were sequenced to 86% average saturation (fig. S1D and data S1).\n",
    "\n",
    "####################\n",
    "ANALYSIS:\n",
    "\n",
    "##########\n",
    "PART 1:\n",
    "- We next performed unbiased clustering of nuclear profiles from ASD and control samples combining both cortical regions (Fig. 1C)\n",
    "- We annotated clusters according to expression of known cell type markers (data S3) and identified 17 cell types, including subtypes of excitatory neurons, interneurons (Fig. 1, D to F), and astrocytes (Fig. 1G).\n",
    "- We observed that neurons expressed more genes and transcripts than glia (fig. S1F).\n",
    "- We tested whether our findings were representative of observations from bulk sequencing studies by performing whole-tissue RNA isolation and sequencing. Bulk gene expression changes in our ASD patient cohort correlated with relative changes from a published RNA-seq dataset (3) (fig. S3E). Our bulk RNA-seq analysis yielded a significant number of common differentially expressed genes (DEGs) (fig. S3F).\n",
    "- We used in situ RNA hybridization to validate the increased density of protoplasmic astrocytes and to confirm the expression of markers for neurogranin (NRGN)–expressing neurons and subtypes of astrocytes (fig. S3, A to D).\n",
    "\n",
    "##########\n",
    "PART 2: (To identify genes dysregulated in ASD in a cell type–specific manner)\n",
    "- To identify genes dysregulated in ASD in a cell type–specific manner, we compared nuclear profiles from ASD and control subjects for each cell type using a linear mixed model (LMM) (see supplementary materials). \n",
    "- We detected 692 differential expression events (q value < 0.05; expression level change ≥ 10%) in 513 unique DEGs, of which 79% (407 of 513) were differentially expressed in a single cell type (data S4). Gene expression changes were of smaller magnitude than expected from the bulk RNA-seq analysis, possibly because of dropout events.\n",
    "- Only 17% of DEGs were among markers of specific cell types, suggesting cell type–specific dysregulation of ubiquitously expressed genes.\n",
    "- The top differentially expressed neuronal genes were down-regulated in L2/3 excitatory neurons and vasoactive intestinal polypeptide (VIP)–expressing interneurons (Fig. 2A). \n",
    "\n",
    "##########\n",
    "DATA-BASE search:\n",
    "SFARI database:\n",
    "- We analyzed the intersection between our gene list and the list of 851 genes that have evidence of genetic association with ASD from the SFARI database (6).\n",
    "Of 513 DEG genes, 75 (13%) were found in the SFARI database (P = 1.9 × 10−17, hypergeometric test) (fig. S3H), including 26 top SFARI genes (Fig. 2C). Additionally, we found significant overlap with high-confidence ASD-associated genetic risk factors (7, 8) (Fig. 2D and fig. S3I). SFARI genes were most overrepresented in L2/3 and L4 excitatory neurons, followed by VIP and somatostatin (SST)–expressing interneurons (Fig. 2E).\n",
    "\n",
    "Gene Ontology (GO, pathway ananlysis)\n",
    "- Gene Ontology (GO) analysis demonstrated that chemical synaptic transmission, axon guidance, neuronal migration, and γ-aminobutyric acid (GABA) signaling were among the top dysregulated pathways (Fig. 2F).\n",
    "- We observed enrichment in similar GO terms when only neuronal DEGs were used (fig. S4A), but no GO terms were identified for glial DEGs, suggesting convergence on the same cellular pathways in neuronal but not glial cell types. Similar pathways were enriched for DEGs with high nuclear RNA–mRNA correlation (Fig. 2, G and H), and DEGs common to several cell types were associated with the development of neuronal projections and adhesion (fig. S4B).\n",
    "##########\n",
    "#end data base search\n",
    "\n",
    "- By downsampling the data to compare the same number of nuclei across all cell types, we found that L2/3 excitatory neurons had the largest number of DEGs (Fig. 2I), followed by L4 excitatory neurons and microglia.\n",
    "- The number of DEGs did not correlate significantly with the number of genes expressed in a given cell type (Pearson r = 0.27, P = 0.3). Thus, our results point to enrichment of ASD molecular changes in cellular components of upper-layer cortical circuits.\n",
    "\n",
    "##########\n",
    "PART 3: (investigated specific gene expression changes)\n",
    "- We then investigated specific gene expression changes in neuronal (Fig. 3A) and glial subtypes.\n",
    "- Hierarchical clustering of cell types based on relative changes of gene expression between ASD and control samples (Fig. 3B) revealed that cell types clustered on the basis of developmental lineage.\n",
    "- Among the top genes dysregulated in L2/3 and L4 neurons, we observed genes important for synaptic function, such as STX1A, SYN2, and NRXN1, as well as the transcription factors TCF25, SOX5, and RBFOX3, which are crucial for brain development (Fig. 3, C and D). TCF25 was also down-regulated in VIP interneurons, as was the transcription factor AHI1 and the synaptic gene RAB3A (fig. S4C). \n",
    "- Microglia from ASD samples were enriched for genes associated with microglial activation and transcriptional factors regulating developmental processes (fig. S4D). We observed similar dysregulation of developmental transcription factors in protoplasmic astrocytes, as well as up-regulation of cell motility (fig. S4E).\n",
    "- Deconvolution of bulk transcriptomic data from ASD patients (supplementary text) suggested that astrocytes in ASD are in an activated state and dysregulate genes necessary for amino acid transport (fig. S4, F and G).\n",
    "- A number of genes were dysregulated in a region-dependent manner (fig. S4H and data S4) and were enriched in processes associated with neuron differentiation and cell migration (Fig. 3E). These included genes in L2/3 neurons, L4 neurons (Fig. 3, F and G), and interneurons (fig. S4I). \n",
    "- Overall, our data point to dysregulated development and synaptic signaling in components of upper-layer cortical circuitry, as well as changes in the cellular state of microglia and protoplasmic astrocytes.\n",
    "\n",
    "##########\n",
    "PART 4:\n",
    "- We next tested whether cell type–specific changes correlated with the clinical severity of ASD.\n",
    "- We obtained Autism Diagnostic Interview–Revised (ADI-R) scores that measured impairment of behavioral domains in ASD (Fig. 4A). We correlated patient ADI-R scores with relative changes of DEGs and observed that changes in L2/3 neurons and microglia were the most predictive of clinical severity (Fig. 4B).\n",
    "- We found that patients with different degrees of clinical severity clustered according to dysregulation of L2/3 neuron and microglia DEGs (Fig. 4, C and D). \n",
    "- Genes most correlated with clinical severity (fig. S5, A and B) were not among the top ASD DEGs, which suggests that the degree of dysregulation is not an accurate predictor of correlation with clinical symptoms.\n",
    "- We next compared single-cell profiles from individual ASD patients to control profiles across all clusters to identify cell types that are recurrently affected across multiple patients (Fig. 4E and fig. S5, C and D). These included upper (L2/3) and deep layer (L5/6-CC) cortico-cortical projection neurons, whereas corticofugal L5/6 projection neurons were not enriched in ASD DEGs. \n",
    "- These results highlight the convergence of ASD-associated changes on cortico-cortical projection neurons across layers.\n",
    "- Furthermore, whole-exome sequencing identified potentially deleterious genetic variants that correlated with reduced expression of corresponding genes (data S5), suggesting a potential link between genomic variations and transcriptional dysregulation in individual ASD patients.\n",
    "\n",
    "##########\n",
    "PART 5:\n",
    "- Because epilepsy is often a comorbidity of ASD, we analyzed shared and divergent molecular changes in ASD and sporadic epilepsy.\n",
    "- We analyzed snRNA-seq data from PFC samples from eight epilepsy patients and seven matched controls (fig. S6, A to C).\n",
    "- We found that epilepsy shared only 10% of all identified cell type–specific gene expression changes we observed in ASD, including 20% of the genes dysregulated in L2/3 neurons (fig. S6, D to F). \n",
    "- GO analysis of DEGs detected in ASD, but not epilepsy, identified synaptic transmission, axon guidance, and brain development as enriched pathways (fig. S6G).\n",
    "- Changes associated with epilepsy comorbidity in the ASD group were enriched in L5/6 corticofugal projection neurons and parvalbumin interneurons (fig. S6H).\n",
    "- Therefore, the majority of molecular changes and core dysregulated pathways we identify in ASD samples are the result of primary ASD pathogenesis and not seizure activity.\n",
    "\n",
    "##########\n",
    "PART 6:\n",
    "- Previous studies suggested convergence of ASD on specific cell types during fetal development (9, 10). \n",
    "- We found that cell types with shared developmental lineages exhibit convergent transcriptional changes in adult ASD patients, and that the expression of synaptic and neurodevelopmental genes in layer 2/3 cortical neurons is especially affected.\n",
    "- This implies that disturbances of gene regulatory programs during development cascade into molecular pathology in specific mature neural cell types.\n",
    "\n",
    "\n",
    "\n",
    "####################\n",
    "Findings/Results:\n",
    "ABSTRACT:\n",
    "- synapticsignaling of upper-layer excitatory neurons and the molecular state of microglia arepreferentially affected in autism.\n",
    "- results show that dysregulation ofspecific groups of genes in cortico-cortical projection neurons correlates with clinicalseverity of autism\n",
    "\n",
    "TEXT:\n",
    "- Our results show that specific sets of genes in upper-layer projection neurons and microglia correlate with the clinical severity of ASD.\n",
    "- By analyzing a cohort of patients with sporadic epilepsy, we were able to disentangle seizure-associated and primary ASD–associated gene expression changes.\n",
    "- The ASD-specific genes highly correlated with clinical phenotypes represent high-priority therapeutic targets for ASD.\n",
    "- Future studies involving larger patient cohorts, including whole-exome sequencing and improved single-cell technologies, will allow for more precise identification of ASD-driven molecular changes and their association with deleterious genetic variants.\n",
    "- Our data describing transcriptomic changes across cortical cell types in ASD, highlighting the most implicated cell types, genes, and pathways, can be interrogated though an interactive web browser: https://autism.cells.ucsc.edu (11).\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "####################\n",
    "Conclusions:\n",
    "- findings suggest that molecular changes in upper-layercortical circuits are linked to behavioral manifestations of autism\n",
    "\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "THE DATA SET\n",
    "Single-Cell Analysis of Autism Spectrum Disorder Implicates Upper Layer Neurons and Protoplasmic Astrocytes as Disease Targets\n",
    "\n",
    "We aimed to gain insight into the molecular pathology of specific neuronal and glial subtypes by analyzing single cells in the cortex of ASD patients. To this end, we performed unbiased nuclei isolation and snRNA-seq 48 post-mortem tissue samples from the prefrontal (PFC), anterior cingulate (ACC) and insular (IC) cortical regions of 16 patients with ASD and 16 control subjects followed by high-throughput single-nucleus RNA sequencing.\n",
    "We generated more that 120,000 single-nuclei RNA-seq profiles and identified 11 neuronal and six glial cell types; these included known subtypes of excitatory neurons and interneurons, main types of glial cells and brain endothelial cells. To identify genes dysregulated in ASD in a cell type-specific manner, we compared nuclei profiles from ASD samples and control subjects. In total, we identified 707 differentially expressed genes (DEGs); 81% of these genes were differentially expressed in a single cell type.\n",
    "Our findings show that development and synaptic signaling of upper-layer excitatory neurons is especially affected in ASD. Moreover, we find significant dysregulation of grey matter astrocyte-encoded genes responsible for astrocyte development and cell-cell signaling. These findings indicate molecular pathological changes in upper-layer cortical neuron circuits and aberrant astroglia-neuronal interactions in ASD and identify potential cellular and molecular therapeutic targets.\n",
    "\n",
    "\n",
    "Lab: Kriegstein\n",
    "Publication: Velmeshev et al. 2019. Science.\n",
    "PubMed Abstract: 31097668\n",
    "NCBI Bioproject: PRJNA434002\n",
    "Submitted by: Dmitry Velmeshev (2019-10-18), Version 3\n",
    "Direct link to this plot for manuscripts: https://autism.cells.ucsc.edu\n",
    "Dataset classification: Organs=brain; Diseases=Autism Spectrum Disorder,Healthy Control; Organism=Human (H. sapiens);\n",
    "Cell Browser dataset ID: autism\n",
    "\n",
    "\n",
    "WHAT IS CONTAINED IN THE DATA FILES?\n",
    "\n",
    "Matrix: exprMatrix.tsv.gz\n",
    "Values in matrix are: 10x UMI counts from cellranger, log2-transformed\n",
    "\n",
    "Raw count matrix: rawMatrix.zip\n",
    "Matrix with raw read counts, includes a copy of the meta data\n",
    "\n",
    "Help: Load matrix/meta into Seurat or Scanpy\n",
    "\n",
    "Cell meta annotations: meta.tsv\n",
    "\n",
    "Dimensionality reduction coordinates:\n",
    "tMinusSNE.coords.tsv.gz\n",
    "\n",
    "Dataset description: desc.json\n",
    "\n",
    "Cell Browser configuration: dataset.json\n",
    "\n",
    "HOW TO LOAD DATA:\n",
    "https://cellbrowser.readthedocs.io/en/master/load.html\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.11.3"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
